src/algorithms/mma_algorithm.jl

```diff
@@ -245,7 +246,6 @@ function optimize!(workspace::MMAWorkspace)
             # Reduces the chance of overflow or underflow in λs 
             # Helps with the convergence
             auto_scale && iter < 5 && scaleobjective!(model, fg, ∇fg, approxfg, λ)
-            #iter < 10 && scaleobjective!(model, fg, ∇fg, approxfg, λ)
```

Comment:
please don't remove this line

Reply:
this file is not related to this PR

Reply:
> this file is not related to this PR

I thought of it as my unused debugging command...

Reply:
actually ya it's useless, it' fine, let's get rid of it